server/fishtest/views.py

```diff
@@ -490,6 +490,19 @@ def get_master_bench():
     return None
 
 
+def get_valid_books():
+    response = requests.get(
+        "https://api.github.com/repos/official-stockfish/books/contents"
+    ).json()
+    return list(
```

Comment:
In modern python an iterable comprehension (or in this case a memory saving generator expression) is replacing the use of functional programming elements such as `map`, `filter`, see
https://stackabuse.com/functional-programming-in-python/

`endswith()` accepts a tuple of strings to be searched for
https://docs.python.org/3/library/stdtypes.html?highlight=endswith#str.endswith

```python
    books_list = (
        b["path"].replace(".zip", "")
        for b in response
        if b["path"].endswith((".epd.zip", ".pgn.zip"))
    )
    return books_list
```

DEV running with this change.